packages/perseus/src/widgets/table/table.tsx

```diff
@@ -138,31 +138,22 @@ class Table extends React.Component<Props> implements Widget {
     focusInputPath(path: FocusPath): void {
         const inputID = getRefForPath(path as Path);
         const inputComponent = this.answerRefs[inputID];
-        if (this.props.apiOptions.customKeypad) {
-            inputComponent.focus();
-        } else {
-            // @ts-expect-error - TS2531 - Object is possibly 'null'. | TS2339 - Property 'focus' does not exist on type 'Element | Text'.
-            ReactDOM.findDOMNode(inputComponent).focus();
-        }
```

Comment:
This is the section of code that was able to be cleaned up by converting SimpleKeypadInput into the forward ref functional component